packages/core/src/sessionInfo/ISessionInfo.ts

```diff
@@ -74,4 +74,9 @@ export interface ISessionInternalInfo {
    * The OIDC issuer that issued the tokens authenticating the session.
    */
   issuer?: string;
+
+  /**
+   * The redirect IRI registered when initially logging the session in.
```

Comment:
Pesky URI/URL/IRI naming, but in this case I think it's just URL:
```suggestion
   * The redirect URL registered when initially logging the session in.
```